Maestro/subscriptions.json

```
@@ -828,6 +828,23 @@
         }
       }
     },
+    // Automate opening PRs to merge extensions main branch back to dev
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/extensions/blob/main//**/*"
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
+      "actionArguments": {
+        "vsoSourceBranch": "dev",
```

This is for the branch that the [branch-merge-pr-generation-pipeline] should pull its yaml from. This should be main I think?
```suggestion
        "vsoSourceBranch": "main",
```

---

Oh, great catch, yeah I'm guessing main is right.